banjo/elab-def.cpp

```diff
@@ -39,13 +39,38 @@ Parser::elaborate_definition(Decl& d)
   {
     Parser& p;
     void operator()(Decl& d)          { lingo_unhandled(d); }
+    void operator()(Super_decl& d)    { p.elaborate_super_declaration(d);}
     void operator()(Variable_decl& d) { p.elaborate_variable_initializer(d); }
     void operator()(Function_decl& d) { p.elaborate_function_definition(d); }
     void operator()(Type_decl& d)     { p.elaborate_type_definition(d); }
   };
   apply(d, fn{*this});
 }
 
+void
+Parser::elaborate_super_initializer(Super_decl& d)
+{
+  struct fn
```

Comment:
You don't need to dispatch here since the only viable definition is `Empty_def`.